packages/story-editor/src/components/videoTrim/useVideoNode.js

```diff
@@ -122,7 +122,8 @@ function useVideoNode(videoData) {
     if (isFinite(videoNode.duration) && startOffset === null) {
       onLoadedMetadata({ target: videoNode });
     } else if (!isFinite(videoNode.duration)) {
-      // Video is a blob of unknown length, wait until it finishes
+      // Video is a blob of unknown length, seek to infinity and wait until it's ready
+      videoNode.currentTime = Number.MAX_SAFE_INTEGER;
```

Comment:
Do we need to reset `currentTime` again in the `timeupdate` callback?

Reply:
The video just restarts when looping if current time is set beyond duration, so there's no issue there.